app/assets/stylesheets/layout/_drop_down.sass

```diff
@@ -49,7 +49,6 @@
 .dropdown .dropdown-panel,
 .toolbar .legacy-actions-more
   min-width: 200px
-  max-width: 360px
```

Comment:
Well if we remove the line the menu gets as wide as its widest <li>-element. As far as I know we do not have that large elements at the moment. But this could change and other languages could also widen the elements, too. 
What we could do instead is to allow line breaks, so that at least many words are shown. However this would still cut one very long word. What do you think?